.github/workflows/changesets.yml

```
@@ -0,0 +1,35 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    if: ${{ github.repository_owner == 'cloudflare' }}
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: "latest"
+          cache: "npm"
+
+      - name: Install modules
+        run: npm ci
+
+      - name: Create Version PR
+        id: changesets
+        uses: changesets/action@v1
+        with:
+          publish: npx changeset publish
```

this will run `npm publish`, which we don't want. instead, I think we wanna do:
```suggestion
          createGithubReleases: true
```